bin/gen-api-package

```diff
@@ -267,13 +269,13 @@ var main = function main() {
   if (opts.outDir) {
     opts.outDir = path.resolve(opts.outDir);
   }
-  if (opts.repoDir) {
-    opts.repoDir = path.resolve(opts.repoDir);
-  }
+  opts.repoDirs = opts.repoDirs || [];
```

Comment:
With the defaultValue, `repoDirs` can be an array always and then this line will no longer necessary.